### DISCUSSION TITLE
Spring Actuator health check 여부

### DISCUSSION THREAD

**Discussion by johnny19991006** [BCSDLab/KOIN_API_V2#277] [에러 / 고민사항]
현재는 그냥 was만 health check 하고 있는데, deploy시 다른 process의 health check의 필요성에 대해서 어떻게 생각하시는지?
사실 현재 health check는 많이 진행되고 있기에 큰 필요는 없지만 미연에 방지를 할 수 있는 방법중 하나라고 생각이 듭니다

**Comment by johnny19991006** [comment]
#276 의 2안을 확인해보시면 좋을거 같아요

**Comment by Choi-JJunho** [comment]
datadog 있어서 굳이~ 라는 생각입니다
